README.md

```diff
@@ -1,15 +1,19 @@
 # OpenTelemetry Semantic Conventions
 
-Welcome to the new repository!
+[![Checks](https://github.com/open-telemetry/semantic-conventions/workflows/Checks/badge.svg?branch=main)](https://github.com/open-telemetry/semantic-conventions/actions?query=workflow%3A%22Checks%22+branch%3Amain)
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/open-telemetry/semantic-conventions.svg?logo=opentelemetry&&color=f5a800&label=Latest%20release)](https://github.com/open-telemetry/semantic-conventions/releases/latest)
+[![Specification Version](https://img.shields.io/badge/OTel_specification_version-v1.26.0-blue?logo=opentelemetry&color=f5a800)](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.26.0)
 
-This is currently a direct copy/filter-branch of the Specification repository
-with only semantic conventions included.
+![OpenTelemetry Logo](https://opentelemetry.io/img/logos/opentelemetry-horizontal-color.png)
 
-This repository is currently using [this specification version][SpecificationVersion].
+Semantic Conventions define a contract between the signals that
+instrumentation will provide and analysis tools that consume
+the instrumentation (e.g. dashboards, alerts, queries, etc.).
```

Comment:
We use a different wording here: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md?plain=1#L17
> The Semantic Conventions define a common set of (semantic) attributes which provide meaning to data when collecting, producing and consuming it.

There isn't much semantic difference, but we should try to consistently use the same definition (at least in this repository) to avoid confusion.

Reply:
I think we need to fix things then, because in that part we link to this page in the spec about stability https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#semantic-conventions-stability and there's another definition there, which is the one I copied. 

I will change to the one you mentioned, at least it's consistent here.